### PR TITLE
certificate access 

### PR DESCRIPTION
**Pull request by andreiSaw**

### CODE REVIEW

**Comment by andreiSaw** [comment]
updated @sashabaranov

File: README.md [hunk 1]
```diff
@@ -42,6 +42,6 @@ certstrap request-cert -ip 127.0.0.1
 certstrap sign 127.0.0.1 --CA disneyland
 
 # For client
-certstrap request-cert -o ship-shield --cn test-user
+certstrap request-cert -o ship-shield.ANY.ANY --cn test-user
```
Comment (sashabaranov) [inline README.md]: I'm guessing ANY usage should be elaborated, as it is special case

File: disneyland/server.go [hunk 2]
```diff
@@ -24,30 +24,26 @@ func (s *Server) Init() {
 
 func (s *Server) CreateJob(ctx context.Context, in *Job) (*Job, error) {
 	user := getAuthUserFromContext(ctx)
-	in.Project = user.Project
-
-	created_job, err := s.Storage.CreateJob(in, user)
-	if err != nil {
-		return nil, detailedInternalError(err)
+	if in.Project == "" {
+		in.Project = user.Project
```
Comment (sashabaranov) [inline disneyland/server.go]: Job should be forced to user's project, otherwise user would be able to write to other projects.

**Comment by andreiSaw** [comment]
updated @sashabaranov

**Pull request merged**